admin/webUI/assets/css/custom.css

```diff
@@ -104,7 +104,7 @@ button.btn-lg {
   vertical-align: middle;
 }
 .mdl-button {
-  height: 48px;
+  height: 42px;
```

Comment:
Just know `.mdl-button` is material's class, this change will reflect in every button. Do it only if want to reflect this everywhere.